### PR TITLE
Add drone page

### PR DESCRIPTION
**Pull request by Xanonymous-GitHub**
## Description
Mapbox has been applied.

Note that the speed of data push has been accelerated 10x faster since it takes so long time to draw the flight area.

截圖 2022-12-02 上午4 16 48

### CODE REVIEW

**Comment by Xanonymous-GitHub** [comment]
@ricky9667 wake up

**Review by ricky9667 — COMMENTED** [review]
Edit:

My bad, I forgot to `pnpm i`.

File: simulator/simulator.py [hunk 1]
```diff
@@ -88,7 +88,7 @@ def run_data_generation() -> None:
         data: EnvData = next(generator)
         insert_data(data)
         generated_data_amount += 1
-        time.sleep(1)
+        time.sleep(0.1)
```
Comment (ricky9667) [inline simulator/simulator.py]: [nit]
It seems updating too fast for charts in my opinion.
Maybe we can try to decrease the number of coordinates to make it display better.

**Review by ricky9667 — APPROVED** [review]
LGTM with a nit comment.

File: simulator/simulator.py (hunk 1 above)
Comment (Xanonymous-GitHub) [reply simulator/simulator.py]: Addressed in [544db48](https://github.com/ricky9667/smart_space/pull/22/commits/544db48f3d06d3a745570e602aa36abf2aaabe4b).

**Pull request merged**